lib/ExpensiMark.js

```diff
@@ -346,6 +348,40 @@ export default class ExpensiMark {
         return replacedText;
     }
 
+    /**
+     * Map the block to newline
+     *
+     * @param {String} htmlString
+     *
+     * @returns {String}
+     */
+    mapBlock(htmlString) {
+        const splitText = htmlString.split('[block]');
+        let joinedText = '';
+
+        // delete empty spaces at the end
+        while (splitText.length) {
+            if (splitText[splitText.length - 1].trim().length === 0 && !splitText[splitText.length - 1].match(/\n/)) {
+                splitText.pop();
+            } else {
+                break;
+            }
+        }
+        for (let index = 0; index < splitText.length - 1; index++) {
+            if (splitText[index].trim() || splitText[index].match(/\n/)) {
+                if (splitText[index].match(/[\n|>][>]?[\s]?$/)) {
+                    joinedText += splitText[index];
+                } else {
+                    joinedText += `${splitText[index]}\n`;
+                }
+            }
+        }
+
+        // last div will not map to new line
```

Comment:
Please capitalize comments
```suggestion
        // Last div will not map to new line
```